org_status/org_hosts/github.py

```diff
@@ -21,6 +22,9 @@ def __init__(self, token, group, **kargs):
         self._token = GitHubToken(token)
         self._org = GitHubOrganization(self._token, self._group)
 
+        for repo in self._org.repositories:
+            repo.get_last_commit_date = MethodType(_get_last_commit_date, repo)
```

Comment:
I don't like how you are patching in a method here. We already have support for Gitman and once your https://github.com/ksdme/org-status/pull/34 gets merged we will support `--use-repo-list`. This method will work when the merge happens.

At this point of time, we have two ways to address this problem.
- Provide a static method on `OrgHost` to get the commit dates of a given repo independently.
- We already support tools like Gitman, we can use them to clone the repos and get commit dates from the local copy.

Reply:
@jayvdb Your thoughts?